packages/core-mobile/app/services/meld/consts.ts

```diff
@@ -0,0 +1,22 @@
+export enum SearchProviderCategories {
+  BankLinking = 'BANK_LINKING',
+  CryptoOnramp = 'CRYPTO_ONRAMP',
+  CryptoOfframp = 'CRYPTO_OFFRAMP',
+  CryptoTransfer = 'CRYPTO_TRANSFER',
+  FiatPayments = 'FIAT_PAYMENTS'
+}
+
+export enum SearchProviderStatus {
+  Live = 'LIVE',
+  RecentlyAdded = 'RECENTLY_ADDED',
+  Building = 'BUILDING'
+}
```

Comment:
```suggestion
export enum SearchProviderStatus {
  LIVE = 'LIVE',
  RECENTLY_ADDED = 'RECENTLY_ADDED',
  BUILDING = 'BUILDING'
}
```
let's follow this pattern